app/models/event_log.rb

```diff
@@ -22,6 +23,9 @@ class EventLog < ApplicationRecord
   has_environment
   has_account
 
+  denormalizes :event, from: :event_type, prefix: :event_type
+  denormalizes :event_type_id, to: :request_log
```

Comment:
Possible race condition here since there's no guarantee an event log will be inserted after a request log. Need to rethink this, and write appropriate tests.